src/sage/arith/misc.py

```diff
@@ -1693,8 +1699,8 @@ def plot(self, xmin=1, xmax=50, k=1, pointsize=30, rgbcolor=(0,0,1), join=True,
         EXAMPLES::
 
             sage: from sage.arith.misc import Sigma
-            sage: p = Sigma().plot()
-            sage: p.ymax()
+            sage: p = Sigma().plot()                                            # optional - sage.plot                  # optional - sage.libs.pari
```

Comment:
Why not using `# optional - sage.plot sage.libs.pari` ?

Reply:
Thanks. I think I must have messed up a merge in this file. I already had a better version on #35095. I have now updated from there in 97d240e.